pages/api/labs/questionGenerators/algebraQuestionGenerator.ts

```diff
@@ -0,0 +1,99 @@
+import { QuestionType } from "../../questionTypes";
+import {
+  getRndInteger,
+  getRndTenthsDecimal,
+  getRndHundredthsDecimal,
+  getRandomItemFromArray,
+} from "../../random";
+import { name } from "../../../api/names";
+export type AlgebraSolveQuestion = {
+  questionType: QuestionType.ALGEBRA_SOLVE_VARIABLE;
+  variableLetter: string;
+  variableproblem: string;
+  algebrasolveModel: AlgebraSolveModel;
+  text: string;
+  answer: string;
+  personname: string;
+};
+
+// Ask coach how to do this properly
+
+
+// Ask coach how to declare 
+export function algebrahelper():list{
```

Comment:
Yeah list isn't a valid return type. I think list is something in python.

I would define a new type and return an array of that type.

`MyCustomType[]`

Check the typescript docs on how to define a array type. Or book time with Abhi

Reply:
Thanks for pointing this out! Declared a custom type instead of using a list. 
![image](https://user-images.githubusercontent.com/99484575/191244339-8186d805-c250-418a-8407-c71ac988b368.png)